start-site/src/main/java/io/spring/start/site/extension/dependency/springcloud/SpringCloudContractMavenBuildCustomizer.java

```diff
@@ -57,7 +63,37 @@ public void customize(MavenBuild mavenBuild) {
 		mavenBuild.plugins().add("org.springframework.cloud", "spring-cloud-contract-maven-plugin", (plugin) -> {
 			plugin.version(sccPluginVersion);
 			plugin.extensions();
+
+			if (isKotlin(this.description.getLanguage()) && isSpringCloudVersionAtLeastAfter()) {
+				plugin.dependency("org.springframework.cloud", "spring-cloud-contract-spec-kotlin", sccPluginVersion);
+			}
 		});
+
+		if (isKotlin(this.description.getLanguage()) && isSpringCloudVersionAtLeastAfter()) {
+			mavenBuild.dependencies().add("spring-cloud-contract-spec-kotlin",
+					Dependency.withCoordinates("org.springframework.cloud", "spring-cloud-contract-spec-kotlin"));
+
+			mavenBuild.plugins().add("org.apache.maven.plugins", "maven-compiler-plugin", (plugin) -> {
```

Comment:
Why is it necessary? Is it because of Kotlin?

Reply:
without it, `ContractVerifierTest.java` can not resolve `BaseContractTest.kt`

I copy this configuration from an example at [spring-cloud-contract-samples](https://github.com/spring-cloud-samples/spring-cloud-contract-samples/blob/master/producer_kotlin/pom.xml#L107-L137)